t/02-bowels/20-add.t

```diff
@@ -204,5 +204,15 @@ my $archive = make_dist_archive("$dist=$pkg1;$pkg2");
 }
 
 #-----------------------------------------------------------------------------
+subtest 'Allow dry run add on locked repo' => sub {
+
+    my $t = Pinto::Tester->new;
+    $t->run_ok( 'Lock' => {} );
+    $t->stack_is_locked_ok('master');
+    $t->run_ok( 'Add', { archives => $archive, dry_run => 1 } );
+    $t->registration_not_ok("AUTHOR/$dist/$pkg1/master");
```

Comment:
agreed - didn't get up to speed on all of the Pinto::Tester helpers, also valid for previous change to 21-pull.t, fixing now